src/main/keep-container/keep-content/keep-label-dropdown/keep-label-dropdown.component.html

```diff
@@ -0,0 +1,37 @@
+<div (click)="stopEvent($event)" *ngIf="labelDropdown && (DialogBoxOpen || !selectedNote)" [ngClass]="{ 'label-Dropdown-menu-up': isLabelsExist(note),'label-Dropdown-position':DialogBoxOpen, 'label-selected': isLabelsExist(note) && DialogBoxOpen}"
```

Comment:
please focus on indentation of your code as well.

Reply:
updated